pages/index.tsx

```diff
@@ -15,6 +19,8 @@ interface SSProps {
   markdown: string
 }
 
+const starmapsGithubUrl = 'https://github.com/pln-planning-tools/StarMaps';
```

Comment:
this should point to the User Guide.md directly

```suggestion
const starmapsGithubUrl = 'https://github.com/pln-planning-tools/StarMaps/blob/main/User%20Guide.md';
```